jekyll/_cci2/configuration-reference.md

```diff
@@ -59,15 +59,21 @@ shell | N | String | Shell to use for execution command in all steps. Can be ove
 steps | Y | List | A list of [steps](#steps) to be performed
 working_directory | N | String | In which directory to run the steps. Default: `~/project` (where `project` is a literal string, not the name of your specific project). Processes run during the job can use the `$CIRCLE_WORKING_DIRECTORY` environment variable to refer to this directory. NOTE: Paths written in your YAML configuration file will _not_ be expanded; if your `store_test_results.path` is `$CIRCLE_WORKING_DIRECTORY/tests`, then CircleCI will attempt to store the `test` subdirectory of the directory literally named `$CIRCLE_WORKING_DIRECTORY`, dollar sign `$` and all.
 parallelism | N | Integer | Number of parallel instances of this job to run (default: 1)
-environment | N | Map | A map of environment variable names and variables. (NOTE: These will override any environment variables you set in the CircleCI web interface.)
+environment | N | Map | A map of environment variable names and values.
 branches | N | Map | A map defining rules for whitelisting/blacklisting execution of specific branches for a single job that is **not** in a workflow (default: all whitelisted). See [Workflows](#workflows) for configuring branch execution for jobs in a workflow.
 resource_class | N | String | Amount of CPU and RAM allocated to each container in a job. (Only available with the `docker` executor) **Note:** A paid account is required to access this feature. Customers on paid plans can request access by [opening a support ticket](https://support.circleci.com/hc/en-us/requests/new).
 {: class="table table-striped"}
 
 <sup>(1)</sup> exactly one of them should be specified. It is an error to set more than one.
 
 #### `environment`
-A map of environment variable names and variables (**Note**: These will override any environment variables you set in the CircleCI web interface).
+A map of environment variable names and values.
+These will override any environment variables
+you set in the CircleCI application.
+
+**Note**:
+While the `environment` key expects a map,
+a list will be accepted without failing or producing an error.
```

Comment:
@marcomorain We can update this note when the code is ready (we are updating the doc now because customers noticed this information is inconsistent with the actual behavior--and you can see we had tons of wrong examples, so we need to make it clear now and not just ignore it.)

Reply:
I spoke with @ndintenfass about this, and he and I both agree with @marcomorain.

The reason this was confusing was because the documentation gave conflicting examples, which I've updated in this PR. This should obviate the need for this note.

Reply:
@smart-alek @nathan @marcomorain this is a good improvement, but it doesn't obviate the need for the note. We know that our system accepts a list today and we should document it now that we've made these changes to the examples.